x/vesting/types/clawback_vesting_account.go

```diff
@@ -180,11 +180,7 @@ func (va ClawbackVestingAccount) ComputeClawback(
 	// Remove all unvested periods from the schedule
 	passedPeriodID := va.GetPassedPeriodCount(time.Unix(clawbackTime, 0))
 	newVestingPeriods := va.VestingPeriods[:passedPeriodID]
-
-	newVestingEnd := va.GetStartTime()
```

Comment:
Not sure about why we need this logic to land in this PR

Reply:
improvements from the new SDK release

Reply:
less code to maintain